Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py

```diff
@@ -0,0 +1,611 @@
+                )
+        else:
+            readableOutput = tableToMarkdown("", response)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs_prefix="",
+                outputs=response
+            )
+
+    else:
+        msg = error_response(err_msg)
+        raise DemistoException(msg)
+
+
+def get_report_cmd(client: Client, args: Dict[str, str]):
+    urlSuffix = "/get_report"
+    taskUuid = args.get(SNXResponse.SNX_TASK_UUID_KEY) or None
+    reportFormat = args.get(SNXResponse.SNX_REPORT_FORMAT_KEY) or "json"
+    if reportFormat is None or reportFormat != "html" and reportFormat != "json":
+        raise DemistoException("Invalid value of report file format paramater")
+    if taskUuid is None:
+        raise DemistoException("Task Uuid Parameter value is not found")
+    elif len(taskUuid) != 84:
+        raise DemistoException("Invalid Task Uuid value")
+    else:
+        reportExtn = "." + reportFormat
+        params = create_request_json(args)
+        response, err_msg = client.get_response(urlSuffix, params)
+        if response:
+            if reportFormat == "json":
+                resJson = json.loads(response)
+                contents = parse_response(resJson)
+                title = None
+                headerList = []
+                readableContents = None
+                for header in contents.keys():
+                    headerList.append(header)
+                title = "SecneurX Analysis - Detailed Report of the Analyzed Sample: {}".format(taskUuid)
+                readableContents = format_report_contents(contents)
+                readableOutputs = tableToMarkdown(title, readableContents, headers=headerList, removeNull=True)
+                return CommandResults(
+                    readable_output=readableOutputs,
+                    outputs=readableContents,
+                    outputs_prefix="SnXAnalysis.Report",
+                    raw_response=resJson
+                )
+            else:
+                reportFileName = taskUuid + reportExtn
+                fileContent = fileResult(reportFileName, response)
+                demisto.results(fileContent)
+        else:
+            msg = error_response(err_msg)
+            if err_msg.res.status_code == 404:
+                result = {'error_msg': msg}
+                readableOutputs = tableToMarkdown("SecneurX Analysis - Failed: {}". format(taskUuid), result)
+                return CommandResults(
+                    readable_output=readableOutputs,
+                    outputs_prefix="",
+                    outputs=result
+                )
+            raise DemistoException(msg)
+
+
+def main():
+    apiKey = demisto.params().get("apiKey")
+    baseUrl = urljoin(demisto.params().get("url"), "/api/v1")
+    verifyCertificate = not demisto.params().get("insecure", False)
+    proxy = demisto.params().get("proxy", False)
+    headers = {"api-key": f"{apiKey}"}
```

Comment:
Do you need the f-string?, you can to write like this:

```suggestion
    headers = {"api-key": apiKey}
```